linkis-computation-governance/linkis-engineconn/linkis-engineconn-plugin-server/src/main/java/org/apache/linkis/engineplugin/server/restful/EnginePluginRestful.java

```diff
@@ -237,7 +235,7 @@ public Message uploadEnginePluginBML(
         }
         engineConnResourceService.refreshAll(true, false);
         log.info("{} finished to upload enginePlugin", username);
-        return Message.ok().data("mes", "upload file success");
+        return Message.ok().data("msg", "upload file success");
```

Comment:
This key will be used in linkis-web:

https://github.com/apache/incubator-linkis/blob/8fbdc3abb68b302d2ba97157abd4d283abe86574/linkis-web/src/apps/linkis/module/EnginePluginManagement/index.vue#L340

Reply:
thanks @wForget  for your review, will  address the code later.